include/lo2s/perf/event_attr.hpp

```diff
@@ -59,22 +60,39 @@ enum class Availability
     UNIVERSAL
 };
 
-inline Availability& operator|=(Availability& a, Availability b) noexcept
+enum class EventFlag
```

Comment:
While I liked this at first, In practice it just feels bad. There is this setFlags method taking a whole bunch of flags as parameters, but is always used only with one flag. Then again, all these flags aren't flags, but decode some manipulation to internal attributes. All flags are bunched up in one enum, even though they have nothing to do with each other. Please use member functions instead. And add documentation to the methods, how each of the methods manipulates the event to a certain behavior.